index.js

```diff
@@ -34,3 +34,34 @@ exports.getFan = function(number) {
 exports.getNumberOfFans = function() {
     return binding.GetFans();
 };
+
+exports.getTemp = function(tempKey, units) {
+    if (!tempKey)
```

Comment:
```js
if (typeof tempKey !== 'string')
```
The C++ bindings will crash out if it's not a string.

Reply:
Ah, I didn't know that, but it makes sense once I think about it for a second. (This is my first time writing node bindings.)